wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -677,9 +679,9 @@ The text in the quotation marks (*title text*) should give additional context to
 
 If an image is the sole content of a paragraph, it displays as a centred block. Otherwise, it flows with the surrounding inline text.
 
-Block images with title text display the title text as a caption below the image.
+Block images with title text display the title text as a caption below the image. The caption will not be rendered if there's an [HTML comment](#comments) on the same line.
```

Comment:
```suggestion
Block images with title text display the title text as a caption below the image. Avoid adding [HTML comment](#comments) or any other text on the same line as the image, as this will cause the caption to not be rendered.
```